### ISSUE TITLE
Empty folder in migrations directory throws: "Cannot read property 'files'....

### ISSUE THREAD

**Issue by cgewecke** [trufflesuite/truffle#2628]
## Issue

Cross-reporting from [solidity-coverage 432][1]. This was fixed for the `test` directory in #2419 but seems to affect `migrations` as well. 

Without an empty folder staged for quick access in an emergency, it wont be possible to put something in a folder when the need arises (without first creating a folder).  

## Steps to Reproduce

```
truffle unbox metacoin
mkdir migrations/emptyfolder
truffle test
```

## Actual Results
```
Compiling your contracts...
===========================
> Compiling ./contracts/ConvertLib.sol
> Compiling ./contracts/MetaCoin.sol
> Compiling ./contracts/Migrations.sol
> Compiling ./test/TestMetaCoin.sol

Error: TypeError: Cannot read property 'files' of undefined
    at Object.run (/Users/cgewecke/code/sc-forks/migra-bug/node_modules/truffle/build/webpack:/packages/migrate/index.js:92:1)
Truffle v5.1.0 (core: 5.1.0)
Node v10.15.3
```

## Environment

* Operating System: OSX
* Ethereum client: ganache
* Truffle version (`truffle version`): 5.1.0
* node version (`node --version`): 10.15.3
* npm version (`npm --version`): 

[1]: https://github.com/sc-forks/solidity-coverage/issues/432

**Comment by eggplantzzz** [comment]
Thanks @cgewecke, looks like it's time for another quick patch :)

**Comment by gnidan** [comment]
I believe this is released in v5.1.1! Closing. Let us know if the problem persists.
